docs/book/known_issues.md

```diff
@@ -75,3 +78,23 @@ Issue 9<a id="issue_9"></a>: CnsRegisterVolume API does not validate if the volu
 
 - Impact: One of the PVC using the CNS volume will be usable at any point in time. Usage of any other PVC will lead to attach failures.
 - Workaround: None
+
+Issue 10<a id="issue_10"></a>: The Supervisor Cluster PVC expanded size is lost and cannot be reclaimed in vSphere 7.0U1 patch release.
+
+- Impact: Users might create a PVC and expand the volume in a Tanzu Kubernetes Grid Service. Before the TKGS PVC is attached to a pod, the TKGS is destroyed. In this case, the corresponding Supervisor Cluster PVC can be reused in another TKGS, but the FilesystemResizePending condition is still on this SVC PVC and cannot be expanded by the new TKGS.
```

Comment:
Can you highlight that the file system will not be expanded?

Reply:
Sure.